cpp/ps0003/main (3).cpp

```diff
@@ -1,16 +1,22 @@
-#include<iostream>
+#include <iostream>
 using namespace std;
+
 int main()
 {
+    // prompting user for number 2/2
+    int num;
+    cout << "enter your num \n";
+    cin >> x;
+
+    // checking if the number is odd or even 3/3
+    if (num >= 0)
+    {
+        if (num % 2 == 0)
+            cout << "even \n";
+        else
+            cout << "odd \n";
+    }
     
-    int x;
-    cout <<"enter your x \n";
-    cin >>x;
-    if (x>=0)
-    if (x % 2 == 0)
-    cout <<"even \n";
-    
-    else
-    cout <<"odd \n";
-return 0;
-}
\ No newline at end of file
+    return 0;
+}
+// great work 😎 8/10 ⭐
```

Comment:
- [x] program returns `0` as everything is okay
- [x] program is in the correct path
- [ ] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code